doc/README.org

```diff
@@ -25,7 +25,18 @@ cat ../version
 - [[https://www.quicklisp.org/beta/][Quicklisp]]
 - curl (refer to your system package manager)
   
-** INSTALLATION
+** AUTOMATIC INSTALLATION
+The default implementation is sbcl, if you are using another then set the ~LANG~
+environment variable, for example to use Clozure common lisp:
+#+begin_src lisp
```

Comment:
This is also a shell source block, isn't it?

Reply:
oops yea, fixed